python/scrabble-score/scrabble_score.py

```diff
@@ -1,23 +1,19 @@
 def score(word):
 
-    word = word.lower()
-    letters = {}
-
     scores = {
-        1: ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't'],
```

Comment:
As before, you can iterate over strings and they are much more pleasant for the eye.